### ISSUE TITLE
Email not sending

### ISSUE THREAD

**Issue by wpraas** [waifung0207/ci_bootstrap#9]
Hi,
Thanks for such a nice Project. My question is this I am working on localhost and there is no email generating. I have setup  Mandrill parameters on 
frontend\config\email.php

There is no error since
I have also uncommented this last line on 
frontend\helpers\My_email_helper.php
echo $CI->email->print_debugger();

How to check if parameters are correctly set and email is going or not?

**Comment by waifung0207** [comment]
I want more info regarding this issue, like:
- does it show a blank page ?
- any error message when you open browser's Developer Tool ?
- does it show success or failure when you turn on "echo $CI->email->print_debugger();" ?

**Comment by wpraas** [comment]
It does not show a blank page
No Erros in the Browser's Developer tool
It shows the success message. (in my case I tried to password reterival, 
A email is sent to you to reset your password.)

However, I have fixed this issue as:
filepath: frontend\config\email.php
Line No. 16:
$config['smtp_pass'] = '';

I have put SMTP key value instead of password. It worked fine.
and form was working fine.

But its should display errors when it was not working fine.
